scousepy/stage_1.py

```diff
@@ -66,9 +66,11 @@ def compute_noise(scouseobject):
         stopcount+=1
         specidx+=1
 
-    rms = np.median(rmsList)
-
-    return rms
+    if np.all(~np.isfinite(np.asarray(rmsList))):
```

Comment:
technically, you don't need `np.asarray` here - `np.isfinite` will treat its argument as an array (I *think* this is correct...)